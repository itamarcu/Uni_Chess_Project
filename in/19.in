game_mode 2
start
move <2,E> to <4,E>
move <7,E> to <5,E>
move <2,F> to <4,F>
move <5,E> to <4,F>
move <2,B> to <3,B>
move <8,D> to <4,H>
move <2,G> to <3,G>
move <4,F> to <3,G>
move <2,H> to <3,H>
undo
move <8,E> to <7,E>
move <3,B> to <4,B>
undo
undo
quit
