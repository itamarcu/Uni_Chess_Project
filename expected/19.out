 Chess
-------
Specify game settings or type 'start' to begin a game with the current settings:
Game mode is set to 2-player
Starting game...
8| R N B Q K B N R |
7| M M M M M M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ _ _ _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m m m m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
8| R N B Q K B N R |
7| M M M M M M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ _ m _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m _ m m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (black player):
8| R N B Q K B N R |
7| M M M M _ M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ M _ _ _ |
4| _ _ _ _ m _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m _ m m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
8| R N B Q K B N R |
7| M M M M _ M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ M _ _ _ |
4| _ _ _ _ m m _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m _ _ m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (black player):
8| R N B Q K B N R |
7| M M M M _ M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ _ m M _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m _ _ m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
8| R N B Q K B N R |
7| M M M M _ M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ _ m M _ _ |
3| _ m _ _ _ _ _ _ |
2| m _ m m _ _ m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (black player):
Check: white king is threatened
8| R N B _ K B N R |
7| M M M M _ M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ _ m M _ Q |
3| _ m _ _ _ _ _ _ |
2| m _ m m _ _ m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
8| R N B _ K B N R |
7| M M M M _ M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ _ m M _ Q |
3| _ m _ _ _ _ m _ |
2| m _ m m _ _ _ m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (black player):
8| R N B _ K B N R |
7| M M M M _ M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ _ m _ _ Q |
3| _ m _ _ _ _ M _ |
2| m _ m m _ _ _ m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
8| R N B _ K B N R |
7| M M M M _ M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ _ m _ _ Q |
3| _ m _ _ _ _ M m |
2| m _ m m _ _ _ _ |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (black player):
Undo move for white player: <3,H> -> <2,H>
Undo move for black player: <3,G> -> <4,F>
8| R N B _ K B N R |
7| M M M M _ M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ _ m M _ Q |
3| _ m _ _ _ _ m _ |
2| m _ m m _ _ _ m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (black player):
8| R N B _ _ B N R |
7| M M M M K M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ _ m M _ Q |
3| _ m _ _ _ _ m _ |
2| m _ m m _ _ _ m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
8| R N B _ _ B N R |
7| M M M M K M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ m _ _ m M _ Q |
3| _ _ _ _ _ _ m _ |
2| m _ m m _ _ _ m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (black player):
Undo move for white player: <4,B> -> <3,B>
Undo move for black player: <7,E> -> <8,E>
8| R N B _ K B N R |
7| M M M M _ M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ _ m M _ Q |
3| _ m _ _ _ _ m _ |
2| m _ m m _ _ _ m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (black player):
Undo move for white player: <3,G> -> <2,G>
Undo move for black player: <4,H> -> <8,D>
8| R N B Q K B N R |
7| M M M M _ M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ _ m M _ _ |
3| _ m _ _ _ _ _ _ |
2| m _ m m _ _ m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (black player):
Exiting...
